Chainlink

```diff
@@ -0,0 +1,236 @@
+
+The Reputation System proposed for ChainLink would record and publish user ratings of oracle providers and nodes, offering a means for users to evaluate oracle performance holistically. Validator System reports are likely to be a major factor in determining oracle reputations and placing these reputations on a firm footing of [trust.](https://link.smartcontract.com/whitepaper)
+
+**But what about the data?**
+
+The data feeds will be managed by nodes who connect to external APIs and API aggregators like [Fiews.io](https://fiews.io/), [CLC Group](http://www.clcg.io/) and[ LinkPool.io](http://www.linkpool.io/).  LinkPool provides a simple alternative for running your own personal node. Their aim is to allow people to easily earn passive income via effortless staking, and also ease the amount of technical knowledge and experience required for running their own node. 
+
+They achieve this by offering an easy to use and[ intuitive staking dApp](https://staking.linkpool.io/) and also a more hands-on node as a service solution (“NaaS”).  LinkPool offers two main ways to stake using their services: regular staking, which can be done in their intuitive dApp, or Node as a Service (NaaS). The latter is a more hands-on approach for node operation, but without the need of technical understanding on how to build and manage infrastructure. 
+
+The key points for NaaS are:
+
+- LinkPool is aiming for NaaS to be launched at the same time of regular staking.
+- NaaS will have a flat fixed price in USD, billed monthly.
+- You’ll be able to pay in ETH or any ERC20.
+
+LinkPool simply provides a brand-new node, how you manage your identity on the network and the data you provide is up to you. More information can be found in this [Medium post](https://medium.com/linkpool/announcing-node-as-a-service-2034ee42d2d9).
+
+According to LinkPool, their on-chain contracts are now trustless, resulting in a way to safely secure the LINK tokens within a smart contract that retains ownership to the address of the staker rather than transferring ownership to the node.
+
+This means the 3rd party provider in this case — LinkPool, are not holding your tokens at any point.
+
+**Funding**
+
+ChainLink conducted a token distribution with 35% of the distribution going to Node Operators, 35% sold to the public, and 30% retained by the company to fund development.  
+
+![image](https://user-images.githubusercontent.com/17677506/58267786-46324280-7d52-11e9-936f-1274cf79b2f2.png)
+_[Figure 1. Chainlink Token Metrics](https://medium.com/paradigm-fund/chainlink-detailed-review-on-the-project-9dbd5e050974)_ 
+
+**How are funds managed?**
+
+Funds are managed by the Chainlink team.
+
+**How is work other than software development funded?**
+
+Work other than software development is also funded by the development fund managed by Chainlink 
+ 
+**Related projects**
```

Comment:
I would ditch these unless they are related directly to Chainlink - i.e. not just similar

Reply:
These are projects directly integrating with Chainlink.  They are not competitors.